src/gui/gtk.c

```diff
@@ -483,9 +483,9 @@ int dt_gui_gtk_load_config()
   dt_conf_set_int("ui_last/view", DT_MODE_NONE);
   int width = dt_conf_get_int("ui_last/window_w");
   int height = dt_conf_get_int("ui_last/window_h");
-  gint x = max(0, dt_conf_get_int("ui_last/window_x"));
-  gint y = max(0, dt_conf_get_int("ui_last/window_y"));
-  gtk_window_move(GTK_WINDOW(widget), x, y);
+  gint x = dt_conf_get_int("ui_last/window_x");
```

Comment:
There is also `fmax()`, but since these are integers, glib-provided `MAX()`/`MIN()` macro definitely should be enough.